tests/Meilisearch.Tests/VersionTests.cs

```diff
@@ -40,5 +43,16 @@ public void GetSimpleVersionFromCsprojFile()
             Assert.NotNull(value);
             Assert.Equal(versionFromCsproj, value);
         }
+
+        [Fact]
+        public void GetDefaultUserAgentHeader()
+        {
+            var httpClient = new HttpClient();
+            httpClient.AddDefaultUserAgent();
```

Comment:
Your test actually looks like this : 

https://github.com/meilisearch/meilisearch-dotnet/blob/be67632c21475c060a44b63dfeb8c5c3915a8fbe/tests/Meilisearch.Tests/MeilisearchClientTests.cs#L151-L160

You are only testing the `HttpClient` like the test above. You could remove line 155 and it will still works. I'm not sure how to properly test this with a `MeilisearchClient` right now. Let me know if you want to find a solution for this now or open an issue for later.

Reply:
I think it's best to create an issue for later, I think we will change the way we deal with headers btw.